### ISSUE TITLE
[Enhancement]: Add alt text to images in installation guides

### ISSUE THREAD

**Issue by ExperiencersInternational** [PretendoNetwork/website#337]
### Checked Existing

- [X] I have checked the repository for duplicate issues.

### What enhancement would you like to see?

Currently, when installing Pretendo on a Wii U or 3DS, none of the images in the guides seem to contain alt text. Alt text allows people who are visually impaired to access content more easily.

Pretendo strives in being accessible to as many people as possible and currently a small part of the community may be isolated because of lacking alt text.

### Any other details to share? (OPTIONAL)

Partially my fault that this is the case as I contributed to parts of the guide and could have added alt text then.

**Comment by ExperiencersInternational** [comment]
Not aware of any other accessibility issues. Just also looked around at blog articles and some older articles don't seem to have alt text in images.

For example:

https://pretendo.network/blog/9-29-21
